pkg/apis/application/v1alpha1/types.go

```diff
@@ -2350,7 +2352,7 @@ type ResourceActionParam struct {
 	Default string `json:"default,omitempty" protobuf:"bytes,4,opt,name=default"`
 }
 
-// TODO: refactor to use rbacpolicy.ActionGet, rbacpolicy.ActionCreate, without import cycle
+// TODO: refactor to use rbac.ActionGet, rbac.ActionCreate, without import cycle
```

Comment:
```suggestion
// TODO: refactor to use rbac.ActionGet, rbac.ActionCreate, without import cycle
```
Since you've solved the circular dependency issue, can you verify if this is still needs to be done?

Reply:
Yup good point. I'll remove that comment.

Reply:
Actually, looking at this this is still valid. I guess the comment can be amended to 
```suggestion
// TODO: refactor to use rbac.ActionGet, rbac.ActionCreate
```